contents/handbook/people/talent.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Talent
+sidebar: Handbook
+showTitle: true
+---
+
+## Talent Principles 
+
+The talent team is uniquely placed at PostHog to have an outsized influence on the people that join the company in comparison to the rest of the business. This is why we ask our talent partners to think like owners.
+
+PostHog's business model requires us to build more products, to do that we need more engineers. Once we have more products, we require commercial team members to market those products and to look after the customers that sign up. We also need to support those customers and support the growth of the business.
+This means that the people that join PostHog directly impact our growth, hence why we invest so heavily in finding and retaining great people. 
+
+At some companies the Talent team are seen as a support function, at PostHog we view them as a growth function, so we need our talent partners to be thinking about the growth of the business as a whole, not just as a headcount. 
+
+This means taking ownership of understanding what products we need to build, the types of people we need to build those products, understanding how our commercial team make our customers successful and what types of people fit into those commercial roles. Talent partners should also understand how our funnels are working and what is needed to solve any problems within them. 
+
+Talent partners at PostHog do not just interview candidates and put them through, they own the process from beginning to end and should use all their skills to make that process successful. 
+
+## Removing distractions from the rest of the team
+
+As a talent partner, part of the role is to ensure we are removing distractions from the rest of the business where we can. Practically, what this looks like in talent is that other team members really only need to concetrate on assessment and shouldn't need to concern themselves with other areas of the process. 
+
+Throughout the recruitment process there are different ways that talent partners can remove distractions.
+
+## Top of the funnel
+
+Talent partner's should be speaking to hiring managers before a job goes live to understand the types of candidate that they should be looking for at the top of the funnel, this part should continue to be honed as you learn more from feedback at the various stages of the process. We want to avoid passing through inappropriate candidates, that waste time. 
```

Comment:
```suggestion
Talent partners should be speaking to hiring managers before a job goes live to understand the types of candidate that they should be looking for at the top of the funnel, this part should continue to be honed as you learn more from feedback at the various stages of the process. We want to avoid passing through inappropriate candidates, that waste time. 
```